src/rpc/blockchain.cpp

```diff
@@ -895,6 +897,13 @@ UniValue pruneblockchain(const JSONRPCRequest& request)
     }
 
     PruneBlockFilesManual(height);
+
+    // check if we need to change the local service flags in order to signal NODE_NETWORK_LIMITED flags
+    if(g_connman) {
+        ServiceFlags nFlags = g_connman->GetLocalServices();
+        CheckLocalServices(nFlags);
```

Comment:
Why do we need to do this? You can't prune deeper than 128 and NODE_NETWORK is already unset for manually-pruned nodes.